### ISSUE TITLE
System Settings - clicking "Desktop Behavior" makes UI hang

### ISSUE THREAD

**Issue by Tatsh** [RJVB/macstrop#32]
The System Settings UI hangs whenever you click on *Desktop Behavior*. The widget partially loads but clicking on any part of the UI does not render anything. The window title does change if I click another KCM, but there is no way to really get things back to working without restarting. I thought this could be resolved by install kactivitymanagerd which partially solved the issue but did not fix the UI.

<img width="1010" alt="activities -2019-02-07 13-19-24" src="https://user-images.githubusercontent.com/724848/52433405-01eca080-2adb-11e9-82ab-a6fc648a8923.png">

The line I think that is relevant is this:

```
 $ /Applications/MacPorts/KF5/systemsettings5.app/Contents/MacOS/systemsettings5
WARNING: Cannot find style "org.kde.desktop" - fallback: "/opt/local/share/qt5/qml/QtQuick/Controls/Styles/Desktop"
```

This happens when I click on the *Desktop Behavior* option.

**Comment by RJVB** [comment]
Are you still running with hacked ports? If so the first step would be to install all ports (and in particular, KF5 frameworks) as they should be.

Also, please try the Icon view (under the configure icon in the top-left of the systemsettings window.

The presence or not of the activitymanager daemon is not relevant here (it doesn't really work on Mac anyway AFAIK), but you're probably right that is some kind of QML issue. For what it's worth, I just switched to sidebar mode in my equivalent install on Linux and the UI hangs there too. The org.kde.desktop fallback warning is probably unrelated; you'll see that in many places, and the fact that a fallback is used indicates that it is not a fatal issue.

In case the UI hang makes it impossible for you to switch the presentation mode: you can edit ~/.config/systemsettingsrc:

```
[Main]
ActiveView=icon_mode
```

**Comment by Tatsh** [comment]
I undid all my changes, pulled and then ran `port upgrade outdated`.

**Comment by Tatsh** [comment]
This is now fixed in 5.13.5.
